packages/core/src/scenes/Random.ts

```diff
@@ -40,6 +40,28 @@ export class Random {
     return value;
   }
 
+  // This is an optimization.
+  // Since gauss() generates a pair of independent Gaussian random numbers,
+  // it returns one immediately and stores the other for the next call to gauss().
+  private nextGauss: number | null = null;
```

Comment:
I can move this to the top of the file or wherever. I thought it was most clear right above the `gauss()` method.

I would also like to test this but I'm not sure how. I ran `npx lerna run build` but I don't know how to use the built targets in my test project.

Reply:
It should be moved to the very top to follow the structure of other classes:
- static properties
- static constructor
- static methods
- properties
- constructor
- methods

---

The easiest way to test new features is with the `template` package.
Running `npm run template:serve` will start an editor that uses the local builds of all packages.
It's possible to use the builds in an external project using `npm link` but it tends to be cumbersome.